tests/kwio_test.sh

```diff
@@ -104,10 +106,19 @@ function alert_completition_validate_config_file_options_Test
 function alert_completion_visual_alert_Test
 {
   local expected="TESTING COMMAND"
-  configurations["visual_alert_command"]="echo \$COMMAND"
+  configurations["visual_alert_command"]="/bin/echo \$COMMAND"
   ret="$(alert_completion "$expected" "--alert=v")"
   assertEquals "Variable $v should exist." "$ret" "$expected"
   true
 }
 
+function alert_completion_sound_alert_Test
+{
+  local expected="TESTING COMMAND"
+  configurations["sound_alert_command"]="/bin/echo \$COMMAND"
+  ret="$(alert_completion "$expected" "--alert=s")"
+  assertEquals "Variable $s should exist." "$ret" "$expected"
```

Comment:
`$s`? Is that correct?

Reply:
I tried to follow the test done by the `alert_completion_visual_alert_Test` function right above this one. But looking at it now, I can't seem to find any variable named `s`. My bad!
That also makes me wonder why the `alert_completion_visual_alert_Test` function mentions a `v` variable and where it comes from.